test/other/embind_tsgen.d.ts

```diff
@@ -1,3 +1,7 @@
+interface WasmModule {
+  _main(_0: number, _1: number): number;
```

Comment:
As a followup we could try to give better name here perhaps?   Maybe for well known functions like `_main`, `_malloc` and `_free` we could even just hardcode them?